classes/PHPTAL/FileSource.php

```diff
@@ -40,7 +40,11 @@ public function getLastModifiedTime()
 
     public function getData()
     {
-        $content = file_get_contents($this->_path);
+        try {
+            $content = file_get_contents($this->_path);
+        } catch (Throwable $throwable) {
```

Comment:
Wait, since when does this throw? :O

Reply:
🤷
```
1) PhptalTest::testDirAsTemplate
file_get_contents(): read of 8192 bytes failed with errno=21 Is a directory

./PHPTAL/classes/PHPTAL/FileSource.php:43
./PHPTAL/classes/PHPTAL.php:1101
./PHPTAL/classes/PHPTAL.php:826
./PHPTAL/classes/PHPTAL.php:659
./PHPTAL/tests/PhptalTest.php:197
```

Reply:
I've seen this multiple times in other components that were upgraded to PHP 8

Interestingly in other libs I happen to maintain :| 

https://github.com/laminas/laminas-mail/issues/7